### ISSUE TITLE
Oracle Design - ALIAS usage for TIPS implementation

### ISSUE THREAD

**Issue by NicoDuvenage** [mojaloop/design-authority-project#31]
## **Request**:
Discuss the design for the TIPS identifier (ALIAS) oracle

**Artifacts**:
- [ ] Artifact to consider [@Contributor]

**Decision(s)**:
- [ ] Actual decision made as a result of discussion

## **Follow-up**:
- [ ] Alternative actions

**Dependencies**:
- [ ] If Applicable

## **Accountability**:
- Owner: [ @emmakazi ]

**Comment by NicoDuvenage** [comment]
The DA did meet and discussed the above request. It was decided that Emmanuel Kazi would gather more information about the reason why an ALIAS is required and how it will be used. It was pointed out during the discussion that the way it was put forward, it would be a potential security risk and in many instances, not allowed to operate like that as a result of regulatory rules.

This discussion is deferred until further information comes to light from the TIPS representative.

**Comment by emmakazi** [comment]
After a long discussion about the topic in the DA session, I took the matter back to the TIPS team, I presented the DA's concerns that: storing the party details at the switch and disclosing the party details (particularly the transacting account) to the payer DFSP would jeopardize the whole idea behind use of ALIAS and pose some security concerns. 

After scrutiny among TIPS teams it was decided to proceed with the implementation that aligns with the current implementation of Mojaloop API specs. And therefore this matter is put to rest.
